app/views/amendments/browse_section.html.erb

```diff
@@ -1,27 +1,32 @@
 <div>
   <% if section %>
-    <%= browse_title(section, browse_section_amendment_path) %>
+    <% if params['locked_section'] == 'false' %>
+      <%= reset_link(browse_section_amendment_path(locked_section: params['locked_section'])) %>
```

Comment:
mm, you shouldn't need to keep locked_section here.
also, you can use inline unless here:

<%= reset_link(browse_section_amendment_path) unless locked_section? %>